qa/tasks/s3tests.py

```diff
@@ -46,11 +46,12 @@ def download(ctx, config):
         else:
             log.info("Using branch '%s' for s3tests", branch)
         sha1 = cconf.get('sha1')
+        git_remote = cconf.get('git_remote', None) or teuth_config.ceph_git_base_url
         ctx.cluster.only(client).run(
             args=[
                 'git', 'clone',
                 '-b', branch,
-                teuth_config.ceph_git_base_url + 's3-tests.git',
+                git_remote + 's3-tests.git',
```

Comment:
yeah I have used that to test my tagging PR as well once just to make sure that the special s3tests branch passed in teuthology too.. here is a seperate commit that adds that https://github.com/ceph/ceph/pull/15115/commits/9944aad3a4e654447767c919a0d1b2d2eea2ce86